10/alpine/Dockerfile

```diff
@@ -2,20 +2,19 @@ FROM alpine:3.9
 
 ENV NODE_VERSION 10.15.3
 
-RUN addgroup -g 1000 node \
-    && adduser -u 1000 -G node -s /bin/sh -D node \
-    && apk add --no-cache \
-        libstdc++ \
-    && apk add --no-cache --virtual .build-deps \
-        binutils-gold \
-        curl \
-        g++ \
-        gcc \
-        gnupg \
-        libgcc \
-        linux-headers \
-        make \
-        python \
+RUN ARCH= && alpineArch="$(arch)" \
+  && case "${alpineArch##*-}" in \
```

Comment:
Is this needed? x64 is the only option so this can be simplified be removing the alpine specific case statement

Reply:
We need to detect the arch anywys because the other arch will still need to build from source